app/views/pages/home.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>Welcome to my Home page</h2>
```

Comment:
Lets keep it as an `<h1>`

Reply:
@santiagosan93 Thank you for the feedback, it's updated!